com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs

```diff
@@ -1,6 +1,9 @@
 using System;
 using System.Collections.Generic;
 using System.ComponentModel;
+using System.Threading;
```

Comment:
I assume this and the one below are leftovers from the experiments?

Reply:
yes will fix this